traefik/tests/rbac-config_test.yaml

```diff
@@ -638,11 +638,76 @@ tests:
               - get
               - list
               - watch
-  - it: should provide expected cluster rbac when k8s gw api is enabled
+
+  - it: should provide expected cluster rbac when k8s gw api is enabled <v3.0.0
     chart:
-      appVersion: v3.1.0
+      appVersion: v2.11.0
```

Comment:
v28+ does not support traefik v2. 
See https://github.com/traefik/traefik-helm-chart/releases/tag/v28.0.0 and https://github.com/traefik/traefik-helm-chart/blob/master/traefik/templates/requirements.yaml